mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -263,6 +263,27 @@ properties:
                     resource is a recursive structure; when the depth is 2, the CodeSystem table will have a column called
                     concept.concept but not concept.concept.concept. If not specified or set to 0, the server will use the default
                     value 2. The maximum depth allowed is 5.
+                - !ruby/object:Api::Type::NestedObject
+                  name: lastUpdatedPartitionConfig
+                  description: |
+                    The configuration for exported BigQuery tables to be partitioned by FHIR resource's last updated time column.
+                  properties:
+                    - !ruby/object:Api::Type::Enum
+                      name: type
+                      description: |
+                        Type of partitioning.
+                      default_value: :PARTITION_TYPE_UNSPECIFIED
+                      values:
+                        - :PARTITION_TYPE_UNSPECIFIED
+                        - :HOUR
+                        - :DAY
+                        - :MONTH
+                        - :YEAR
+                    - !ruby/object:Api::Type::String
+                      name: expirationMs
+                      description: |
+                        Number of milliseconds for which to keep the storage for a partition.
```

Comment:
Can one of these be required to avoid allowing the user to specify an empty block.

ie, avoiding
```hcl
        last_updated_partition_config {}
```

Reply:
Technically user can create an empty last_updated_partition_config and pass it to api, although they will get an `InvalidArgument` error if they do so. I am not sure if we should allow empty object here so that it behaves the same as our API.

Reply:
I'm not sure we send anything on the terraform side if the block is empty. Unless you set the send_empty_object flag on the nested object

Reply:
Ultimately can these fields be set without the other?

Reply:
I had a talk with our team. Our documentation(https://cloud.google.com/healthcare-api/docs/reference/rest/v1/projects.locations.datasets.fhirStores#streamconfig) needs some improvements because it does not have `required` for all the required fields. For `lastUpdatedPartitionConfig`, the `type` field is actually required. I'll make the change, thanks Scott!